src/index.js

```diff
@@ -211,39 +230,111 @@ exports.init = (Implementation) => {
         });
 
         if (process.env.NODE_ENV !== 'production') {
+          // NOTICE: Reorder the collections to make the diff null or minimal
+          collections = collections.map((collection) => {
+            const collectionReordered = {};
+            _.each(_.sortBy(Object.keys(collection)), (key) => {
+              collectionReordered[key] = collection[key];
+            });
+            collectionReordered.fields = collectionReordered.fields.map((field) => {
+              const fieldReordered = {};
+              _.each(_.sortBy(Object.keys(field)), (key) => {
+                fieldReordered[key] = field[key];
+              });
+              return fieldReordered;
+            });
+            collectionReordered.fields = _.sortBy(collectionReordered.fields, ['field', 'type']);
+            return collectionReordered;
+          });
+          collections.sort((collection1, collection2) =>
+            collection1.name.localeCompare(collection2.name));
+
           const filename = `${path.resolve('.')}/forestadmin-schema.json`;
-          fs.writeFileSync(filename, JSON.stringify(collections, null, 2));
+          const forestAdminSchema = {
+            collections,
+            meta: {
+              database_type: Implementation.getDatabaseType(),
+              liana: Implementation.getLianaName(),
+              liana_version: Implementation.getLianaVersion(),
+              orm_version: Implementation.getOrmVersion(),
+            },
+          };
+          fs.writeFileSync(filename, JSON.stringify(forestAdminSchema, null, 2));
+        }
+
+        if (process.env.FOREST_DISABLE_AUTO_SCHEMA_APPLY
```

Comment:
We have to see with @arnaudbesnier, but he told us a similar variable exists on rails and we should name it the same.

Reply:
I'm not okey with that. The old variable name is missleading

Reply:
Ok, just bring the subject on the standup to have the approval of arnaudb.

Reply:
It's done and deprecated on Rails ;)